content/en/docs/Tasks/creating-an-index.md

```diff
@@ -5,37 +5,123 @@ description: >
   Add/Remove a collection of bundles to/from an Index
 ---
 
-## Prerequisites 
+## Prerequisites
 
-- [opm](https://github.com/operator-framework/operator-registry/releases) `v1.14.0+`
-- [docker](https://docs.docker.com/install/) version `17.03`+ or [podman](https://github.com/containers/libpod/blob/master/install.md) `v1.2.0+` or [buildah](https://github.com/containers/buildah/blob/master/install.md) `v1.7+`.
+- [opm](https://github.com/operator-framework/operator-registry/releases) `v1.19.0+`
 
+## Creating an Index
 
-# Creating an Index 
+`OLM`'s `CatalogSource` [CRD][catalogsource-crd] defines a reference to a catalog of operators that are available to
+install onto a cluster. To make your operator bundle available, you can add the bundle to a container image which the
+`CatalogSource` references. This image contains a record of bundle images that OLM can pull and extract the manifests
+from in order to install an operator.
```

Comment:
This paragraph is a little confusing to me. How about this: 
```suggestion
`OLM`'s `CatalogSource` [CRD][catalogsource-crd] accepts a container image reference to a catalog of operators that can be made available to install onto a cluster. To add your operator bundle to a Catalog (and thereby make your operator bundle available to install in a cluster), you can add the bundle to a Catalog, package the Catalog in a container image, and then use that image reference in the `CatalogSource`. This image contains a record of bundle images that OLM can pull and extract the manifests
from in order to install an operator.
```

Reply:
I agree the original could be re-worded. I took most of what you wrote and made a few more of my own tweaks. Let me know what you think.